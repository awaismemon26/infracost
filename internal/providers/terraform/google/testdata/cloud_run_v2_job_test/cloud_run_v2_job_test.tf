provider "google" {
  credentials = "{\"type\":\"service_account\"}"
  project     = "my-project"
  region      = "europe-west3"
}

resource "google_cloud_run_v2_job" "basic" {
  name = "cloudrun-v2-job-test"
  location = "europe-west4"
  template {
    template {
      containers {
        image = "us-docker.pkg.dev/cloudrun/container/hello"
      }
    }
    task_count = 1
  }
}
