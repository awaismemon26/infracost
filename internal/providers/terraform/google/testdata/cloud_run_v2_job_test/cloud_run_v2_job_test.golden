
 Name                                Monthly Qty  Unit          Monthly Cost   
                                                                               
 google_cloud_run_v2_job.my_job                                                
 ├─ CPU allocation time (tier 2)      30,000,000  vCPU-seconds       $648.00   
 └─ Memory allocation time (tier 2)   15,000,000  GiB-seconds         $36.00   
                                                                               
 OVERALL TOTAL                                                      $684.00 

*Usage costs can be estimated by updating Infracost Cloud settings, see docs for other options.

──────────────────────────────────
1 cloud resource was detected:
∙ 1 was estimated

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━┓
┃ Project                                            ┃ Baseline cost ┃ Usage cost* ┃ Total cost ┃
┣━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╋━━━━━━━━━━━━━━━╋━━━━━━━━━━━━━╋━━━━━━━━━━━━┫
┃ main                                               ┃ $684          ┃ $0.00       ┃ $684       ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┻━━━━━━━━━━━━━━━┻━━━━━━━━━━━━━┻━━━━━━━━━━━━┛